webroot/js/sentences_lists.set_option.js

```diff
@@ -40,9 +40,12 @@
                 rootUrl + "/sentences_lists/set_option/",
                 { "listId": listId, "option": "editable_by", "value": $scope.editable },
                 function () {$(".is-editable.loader-container").hide();}
-            ); 
-            
+            ).then(function success(response) {
+                if(response.reload)
+                    location.reload();
+            });
+
```

Comment:
There is already a callback function up there that does `$(".is-editable.loader-container").hide();`. Could you group the callback code into a single function?